src/olympia/discovery/utils.py

```diff
@@ -52,7 +52,7 @@ def call_recommendation_server(server, client_id_or_guid, data, verb='get'):
         if response.status_code != 200:
             raise requests.exceptions.RequestException()
     except requests.exceptions.RequestException as e:
-        log.error(u'Calling recommendation engine failed: {0}'.format(e))
+        log.exception('Calling recommendation engine failed: %s' % e)
```

Comment:
You want to use the following instead to avoid custom formatting:

```suggestion
        log.exception('Calling recommendation engine failed: %s', e)
```